cmd/server/main.go

```diff
@@ -136,20 +142,45 @@ func (s *Server) getAudienceForDriver(ctx context.Context) (string, error) {
 	return sms.Spec.Audience, nil
 }
 
-func (s *Server) authRequest(ctx context.Context, securityToken, namespace string) error {
+func (s *Server) streamInterceptor(srv interface{}, ss grpc.ServerStream, info *grpc.StreamServerInfo, handler grpc.StreamHandler) error {
+	log.Println("Stream interceptor: performing auth checks for req", info.FullMethod)
+	if err := s.authRequest(ss.Context()); err != nil {
+		return err
+	}
```

Comment:
Should we also be checking if `info. IsServerStream` is true and/or that the names of the method match `GetDelta` or `GetAllocated`?  May be unnecessary and too complicated ....